crates/ruma-client-api/src/r0/account/registration_token_validity.rs

```diff
@@ -0,0 +1,41 @@
+//! [GET /_matrix/client/unstable/org.matrix.msc3231/register/org.matrix.msc3231.login.registration_token/validity](https://github.com/matrix-org/matrix-doc/blob/main/proposals/3231-token-authenticated-registration.md#checking-the-validity-of-a-token)
+
+use ruma_api::ruma_api;
+
+ruma_api! {
+    metadata: {
+        description: "Checks to see if the given registration token is valid.",
+        method: GET,
+        name: "registration_token_validity",
```

Comment:
The (module) name should be an *action*, probably `check_registration_token_validity` (we have two `check_` prefixed routes in identity-service-api too).

Reply:
Ah, ok.